pyproject.toml

```diff
@@ -0,0 +1,40 @@
+[tool.poetry]
+name = "trustpilot"
+version = "8.0.0"
+description = "trustpilot api client including cli tool"
+authors = [
+    "Johannes Valbjørn <jgv@trustpilot.com>",
+    "Thomas Thiebaud <tht@trustpilot.com>"
+]
+readme = "README.md"
```

Comment:
is there support for mentioning the HISTORY.md as well?

Reply:
maybe just link to the history file instead

Reply:
remember to include(license, readme) etc

Reply:
license and readme are included. There is also a link to the repo, but there is no support for `HISTORY.md`